### ISSUE TITLE
Add Automation Scripts

### ISSUE THREAD

**Issue by prathimacode-hub** [prathimacode-hub/Awesome_Python_Scripts#4]
This issue is specifically created for those who have basic programming knowledge in python and want to explore on Automation Scripts

Add basic and advance automation scripts in 'AutomationScripts' folder.

Repository : AutomationScripts
Level : Easy and Medium

Do check if the code you want to contribute is available in this repository
Comment on this issue with the title and details of the script you are going to contribute and what does it do
Start working on that script and before you add make sure it works properly
Add your code to 'AutomationScripts' repository using the contribution guidelines and template mentioned
Create a Pull Request and be patient
Code will be reviewed and if all works perfectly fine, PR will be merged.
If anything else needed from you, we will get back with the response
Make good [PR](https://github.com/prathimacode-hub/Awesome_Python_Scripts/blob/main/CONTRIBUTING_GUIDELINES.md) and [clean code](https://github.com/zedr/clean-code-python).

Any queries, drop me a private message or just mention me over here!. I'll be happy to support and assist you.
Best of luck everyone ❤️

Happy Contribution!

**Comment by Chanakya-Sharma** [comment]
Hey can you assign me this @prathimacode-hub

**Comment by Barsha27** [comment]
hey @prathimacode-hub i would like to work on it but can you tell me if there is any specific automation of tasks you would want me to work on?

**Comment by prathimacode-hub** [comment]
Are you a LGMSOC participant? @Barsha27

**Comment by Barsha27** [comment]
no am not

**Comment by prathimacode-hub** [comment]
Ok, just notify here in comment about the script you are going to contribute. You are good to go. And one more thing, there is no specific requirement, it's your choice to choose the script. @Barsha27

**Comment by Barsha27** [comment]
sure i will notify you here...

**Comment by BMaster123** [comment]
Can I make an automation script that sends an email every time a stock crosses a certain price?

**Comment by prathimacode-hub** [comment]
Yes, you can proceed. Create an issue using the issue template provided and get it assigned. You are free to work once its assigned. @BMaster123
